### PR TITLE
added more text explanation to the tutorial

### LINKED ISSUE #1058: Add tutorial for new features added
# Add tutorial for new features

* pretty print feature for metadata.
* Excel stats 
* to_markdown feature for metadata.
* to_excel_template feature to export table schema as excel.

### PR DESCRIPTION
**Pull request by shashigharti**
* Updated notebook with more textual explanations.

- fixes #1058

### CODE REVIEW

**Review by roll — APPROVED** [review]

**Review by lwinfree — APPROVED** [review]
This looks good to me, thanks @shashigharti! I won't merge until Evgeny has approved it though.

**Comment by shashigharti** [comment]
Thanks @lwinfree!.I think Evgeny has already approved it.

**Pull request merged**
